### PR TITLE
Apply the machine-type of the new equipment.

### PR DESCRIPTION
**Pull request by kmdkuk**
Signed-off-by: kouki <kouworld0123@gmail.com>

### CODE REVIEW

File: pkg/generate-sabakan-machines/main.go [hunk 1]
```diff
@@ -26,14 +26,14 @@ func main() {
 	var machineTypeBoot, machineTypeCS, machineTypeSS, bmcType string
 	switch os.Args[1] {
```
Comment (zoetrope) [inline pkg/generate-sabakan-machines/main.go:27]: Do we need this switch statement?

How about passing the machine type as a cli option?

Comment (kmdkuk) [reply pkg/generate-sabakan-machines/main.go:27]: Yes, it is. That way, We don't have to modify the program every time a new equipment appears.

Comment (kmdkuk) [reply pkg/generate-sabakan-machines/main.go]: @zoetrope Modified to accept machine-type as a command line argument using cobra.
The logic for creating json has not been changed.
Please review again.

File: pkg/generate-sabakan-machines/cmd/root.go [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+	"github.com/spf13/cobra"
+)
+
+var rootCmd = &cobra.Command{
+	Use:          "generate-sabakan-machines CSV",
+	SilenceUsage: true,
+	Short:        "generate machines.json",
+	Long: `Generate machines.json from the CSV file for use with the 'sabactl create' command.
+Example:
+  generate-sabakan-machines CSV --machine-type-boot=r6525-boot-1 --machine-type-cs=r6525-cs-1 --machine-type-ss=r7525-ss-1
```
Comment (zoetrope) [inline pkg/generate-sabakan-machines/cmd/root.go:22]: `CSV` does not look like a csv file.

```suggestion
  generate-sabakan-machines input.csv --machine-type-boot=r6525-boot-1 --machine-type-cs=r6525-cs-1 --machine-type-ss=r7525-ss-1
```

**Review by zoetrope — APPROVED** [review]

**Review by zoetrope — APPROVED** [review]

**Pull request merged**